src/CoreWCF.Primitives/src/CoreWCF/Description/IEndpointAddressForMetadataProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+
+namespace CoreWCF.Description
+{
+    public interface IEndpointAddressForMetadataProvider
+    {
```

Comment:
Using the word "For" in an interface name seems a little off. Maybe switch the subject and function to remove it, e.g. IMetadataEndpointAddressProvider

Reply:
Thanks i was not confortable with the name i choosed. I renamed to `IMetadataEndpointAddressProvider`

Reply:
The 3 hardest problems in software development are 1) Naming, 2) Off by one errors